src/components/Body2.md

```diff
@@ -19,11 +19,17 @@
 ```jsx
 import { COLORS } from './Colors'
 ;<>
+  <Body2.Regular400 color={COLORS.BRAND_DUCKEGG}>
+    Brand Duckegg color
+  </Body2.Regular400>
   <Body2.Regular400 color={COLORS.BRAND_FOREST}>
     Brand Forest color
   </Body2.Regular400>
   <Body2.Regular400 color={COLORS.BRAND_SALAMANDER}>
     Brand Salamander color
   </Body2.Regular400>
+  <Body2.Regular400 color={COLORS.GRAY_SECONDARY}>
```

Comment:
Should I create examples for every available color? If we end up doing this for other components too it could get pretty repetitive.

Reply:
No need.